src/service/Review.ts

```diff
@@ -321,6 +321,24 @@ const claim = async (codeReview: CodeReviewRecord, slackUserId: string): Promise
     };
 };
 
+const assign = async (codeReview: CodeReviewRecord, slackUserId: string): Promise<ReviewActionResult> => {
+    const numberReviewRequired = await getRepositoryNumberOfReviews(extractRepository(codeReview.pullRequestLink));
+    await setCodeReviewerStatus(codeReview, slackUserId, 'pending');
```

Comment:
Do the variables below depend on side effects from this function? if not, does this need to await?

Reply:
Yes it does because of calculateReviewStats need the data to commit first.